languages/go/exercises/concept/using-map/.meta/example.go

```diff
@@ -0,0 +1,46 @@
+package usingmap
+
+import (
+	"errors"
+)
+
+var priceTable = map[string]int{
+	"men_tops":         2,
+	"men_underwares":   1,
+	"men_jeans":        4,
+	"robes":            4,
+	"jackets":          7,
+	"women_tops":       4,
+	"women_underwares": 2,
+	"women_jeans":      3,
+}
+
+// TransactionLog represent a single item in a transaction
+type TransactionLog struct {
+	itemName string
+	month    string
+	qty      int
+}
+
+func calculateTotalIncome(monthlyIncome map[string]int) int {
+	var total int
+
+	for _, monthlyIncome := range monthlyIncome {
+		total += monthlyIncome
+	}
```

Comment:
Hmmm... wondering what we should do now. `range` loop is actually introduced after the `map` and not known at this point. Same below.

Also `slices` are not a prerequisite for this exercise, which are used below...

Reply:
@ErikSchierboom: What do you think? Any ideas?

Reply:
> Hmmm... wondering what we should do now. range loop is actually introduced after the map and not known at this point. Same below.
>
> Also slices are not a prerequisite for this exercise, which are used below...

I think the only logical step is to have the ranges and slices as prerequisites for this exercise, or are the exercises that deal with those Concepts dependent on Maps?

Reply:
ranges are dependant on maps, yes